src/lib/components/workflow/workflow-summary.svelte

```diff
@@ -31,16 +31,14 @@
       $workflowSummaryViewOpen = !$workflowSummaryViewOpen;
     }}
   >
-    <div
-      class="grid-row-3 grid w-full grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-4 xl:w-11/12"
-    >
-      <div class="col-span-1 md:col-span-2">
+    <div class="flex flex-col gap-2 xl:flex-row">
+      <div class="grow overflow-hidden">
         <h3 class="font-medium">{translate('common.workflow-type')}</h3>
         <div class="h-0.5 rounded-full bg-gray-900" />
         <WorkflowDetail content={workflow?.name} copyable />
         <WorkflowDetail content={workflow?.runId} copyable />
       </div>
-      <div class="col-span-1">
+      <div class="grow overflow-hidden">
```

Comment:
What do you think about using `flex-1` instead of `grow` for the type and task queue sections?

```suggestion
      <div class="flex-1 overflow-hidden">
```

Otherwise the task queue section can get cut off if the workflow name is super long ⬇️ 

| Before | After |
|--|--|
|![Screenshot 2023-11-06 at 9 21 05 AM](https://github.com/temporalio/ui/assets/15069288/04da27fc-76aa-4257-a756-407112afb669)|![Screenshot 2023-11-06 at 9 30 25 AM](https://github.com/temporalio/ui/assets/15069288/77375568-e449-4145-9414-5743eb5ccc2e)|

Reply:
It's a balancing act for sure. I think the most likely situation is a workflow type and task queue that are long but not insanely long like that one. In your example, if the Workflow Type was a little shorter in text, the Type would not be truncated and the task queue wouldn't be either (Before with grow). With flex-1, it will still be truncated.

I think I prefer grow because it's a little more flexible on growing with the long text (with the downside of truncating with super long texts, but I think that's less likely).